### PR TITLE
Updating language codes in Language Classification

### PR DESCRIPTION
**Pull request by asparius**
PR to resolve the bug in #1086, updating obsolete language codes

### CODE REVIEW

File: docs/tasks.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ The following tables give you an overview of the tasks in MTEB.
 | [BlurbsClusteringS2S.v2](https://www.inf.uni-hamburg.de/en/inst/ab/lt/resources/data/germeval-2019-hmc.html) (Steffen Remus, 2019) | ['deu'] | Clustering | s2s | [Fiction, Written] | {'test': 2048} | {'test': 23.02} |
 | [BornholmBitextMining](https://aclanthology.org/W19-6138/) | ['dan'] | BitextMining | s2s | [Web, Social, Fiction, Written] | {'test': 500} | {'test': 89.7} |
 | [BrazilianToxicTweetsClassification](https://paperswithcode.com/dataset/told-br) (Joao Augusto Leite and                         Diego F. Silva and                         Kalina Bontcheva and                         Carolina Scarton, 2020) | ['por'] | MultilabelClassification | s2s | [Constructed, Written] | {'test': 2048} | {'test': 85.05} |
+| [BrightRetrieval](https://huggingface.co/datasets/xlangai/BRIGHT) (Su et al., 2024) | ['eng'] | Retrieval | s2p | [Non-fiction] | {'standard': 1334914, 'long': 7048} | {'standard': 800.3994729248476, 'long': 46527.35839954597} |
```
Comment (isaac-chung) [inline docs/tasks.md:55]: This does not seem relevant to the PR. Wondering if this would go away if we merge updates from `main`?

Comment (asparius) [reply docs/tasks.md:55]: @isaac-chung As you can see from the commit logs, it is done by the github actions. It comes back even if I deleted it.

File: mteb/tasks/Classification/multilingual/LanguageClassification.py [hunk 2]
```diff
@@ -5,26 +5,26 @@
...
-    "pl": ["pol-Latn"],
-    "pt": ["por-Latn"],
-    "ru": ["rus-Cyrl"],
-    "sw": ["swa-Latn"],
-    "th": ["tha-Thai"],
-    "tr": ["tur-Latn"],
-    "ur": ["urd-Arab"],
-    "vi": ["vie-Latn"],
-    "zh": ["cmn-Hans"],
+    "ara": ["ara-Arab"],
```
Comment (KennethEnevoldsen) [inline mteb/tasks/Classification/multilingual/LanguageClassification.py:8]: This is still an incorrect format. "ara" or "ar" refer to the subset on huggingface. However there is no subset on HF only the "default". Thus _LANGUAGES should either be a list [LANG1, LANG2,...] or a dict {"default": [LANG1, LANG2, ...}

**Review by KennethEnevoldsen — CHANGES_REQUESTED** [review]

**Comment by KennethEnevoldsen** [comment]
This dataset still seems incorrectly formatted (the language . I have

**Comment by KennethEnevoldsen** [comment]
Superseded by https://github.com/embeddings-benchmark/mteb/pull/1094
